docker/foo/Dockerfile

```diff
@@ -0,0 +1,45 @@
+FROM arm64v8/centos:7.9.2009
```

Comment:
I guess you didn't mean to add this file.

Reply:
nope... :(